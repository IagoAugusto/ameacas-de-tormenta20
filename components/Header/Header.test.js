import { render } from "@testing-library/react";
import Home from ".";

it("renders header", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
