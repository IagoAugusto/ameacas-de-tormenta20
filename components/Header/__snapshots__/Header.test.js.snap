// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<div>
  <nav
    class="bg-gray-800"
  >
    Criar Ameaças
  </nav>
</div>
`;

exports[`renders homepage unchanged 1`] = `
<div>
  <nav
    class="bg-gray-800"
  >
    Criar Ameaças
  </nav>
</div>
`;
